src/index.js

```diff
@@ -56,51 +56,51 @@ const App = () => {
         return (
           <BrowserRouter>
             <Localization messages={localizationBundle}>
+              <WrappedRequest url={data.instructor_url}>
+                {({data}) => {
+                  const instructor = data
```

Comment:
Moved this to use `instructor.avatar_url` everywhere instead of the previous mix of `user.avatar_url`